lib/inspec/schema/schema.rb

```diff
@@ -0,0 +1,64 @@
+require "json"
```

Comment:
this file should be named `lib/inspec/schema/output_schema.rb`, no?

Reply:
Resolved as of https://github.com/inspec/inspec/pull/4330/commits/a25eabb6d2b7289edc3fccd6771b17767e2fc306